k8s/cloud/testing/kustomization.yaml

```diff
@@ -3,20 +3,20 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: plc-testing
 replicas:
-- name: api-server
-  count: 3
-- name: auth-server
-  count: 3
-- name: profile-server
-  count: 3
-- name: cloud-proxy
-  count: 3
-- name: project-manager-server
-  count: 3
-- name: vzmgr-server
-  count: 1
-- name: scriptmgr-server
-  count: 3
+- count: 3
+  name: api-server
+- count: 3
+  name: auth-server
+- count: 3
+  name: profile-server
+- count: 3
+  name: cloud-proxy
+- count: 3
+  name: project-manager-server
+- count: 1
+  name: vzmgr-server
+- count: 3
+  name: scriptmgr-server
```

Comment:
nit: I personally find these easier to parse as name followed by count instead of the other way around, even though the keys are ascending in the rewritten version.
(ditto for all the sections where this changed.) I'll let you decide if you want to revert to the old version

Reply:
I agree that the former writing parsed better. This was unfortunately the default formatting from `kustomize edit fix`.

I'll revert this section back to it's previous form.